returnn/compile.py

```diff
@@ -203,3 +205,63 @@ def run(self):
             shutil.move(files[0], self.out_op.get_path())
         if len(files) > 1:
             shutil.move(files[1], self.out_grad_op.get_path())
+
+
+class OnnxExportJob(Job):
+    """
+        Export an ONNX model using the appropriate RETURNN tool script.
+
+        Currently only supports PyTorch via tools/torch_export_to_onnx.py
+    """
+
+    def __init__(
+        self,
+        *,
+        returnn_config: ReturnnConfig,
+        checkpoint: Union[Checkpoint, PtCheckpoint],
```

Comment:
Now that I fixed the issue raised by @vieting about `get_model` with `epoch` and `step`, I realize that those should also be arguments here, because the generated model can be dependent on it.

We might want to use those defaults. In many cases, I assume that `get_model` does actually not use it, so such defaults are fine then?

Or do we want to automatically infer it from the checkpoint somehow? But this was based on heuristics, and not sure if this is wanted.

```suggestion
        checkpoint: Union[Checkpoint, PtCheckpoint],
        epoch: int = 1,
        step: int = 0,
```

Reply:
>  Or do we want to automatically infer it from the checkpoint somehow? 

Why infer? we store this very explicitly in the checkpoints already, see:
https://github.com/rwth-i6/returnn/blob/master/returnn/torch/engine.py#L613

Reply:
So I am against having those as extra parameters for the job.

Reply:
Ah yes I forgot. Then I will update the script again. Then it does not need those arguments.

Reply:
Done.